common/scripts/configs/credentials.template

```diff
@@ -0,0 +1,3 @@
+[default]
+aws_access_key_id={{aws_access_key}}
```

Comment:
in fact, if the admiral env file is calling it ACCESS_KEY, then it should be `{{ACCESS_KEY}}` here too